test/images/volumes-tester/nfs/Dockerfile

```diff
@@ -12,16 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos
-MAINTAINER Jan Safranek, jsafrane@redhat.com
-RUN yum -y install /usr/bin/ps nfs-utils && yum clean all
+FROM fedora:24
+LABEL maintainers="Jan Safranek, jsafrane@redhat.com; Matthew Wong, mawong@redhat.com"
+
+# install nfs-ganesha
+RUN dnf install -y tar gcc cmake autoconf libtool bison flex make gcc-c++ krb5-devel dbus-devel && dnf clean all \
+    && curl -L https://github.com/nfs-ganesha/nfs-ganesha/archive/V2.4.0.3.tar.gz | tar zx \
+    && curl -L https://github.com/nfs-ganesha/ntirpc/archive/v1.4.1.tar.gz | tar zx \
+    && rm -r nfs-ganesha-2.4.0.3/src/libntirpc \
+    && mv ntirpc-1.4.1 nfs-ganesha-2.4.0.3/src/libntirpc \
+    && cd nfs-ganesha-2.4.0.3 \
+    && sed -i '20i#cmakedefine _NO_PORTMAPPER 1' src/include/config-h.in.cmake \
+    && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_CONFIG=vfs_only -D_NO_PORTMAPPER=ON src/ \
```

Comment:
I `dnf remove` the dependencies in the last line of this RUN step, so the resulting container won't have all those dependencies I pulled in just for compilation.

Another thing I could do to reduce the size is delete the sources after ganesha has been built, will work on that change now.